api/Animation.json

```diff
@@ -243,6 +243,58 @@
           }
         }
       },
+      "commitStyles": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Animation/commitStyles",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "Currently Chrome Canary only"
+            },
+            "chrome_android": {
+              "version_added": false,
+              "notes": "Currently Chrome Canary only"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "75"
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "13.1"
+            },
+            "safari_ios": {
+              "version_added": "13.4",
+              "partial_implementation": true,
+              "notes": "Passes 13/27 <a href='https://wpt.fyi/results/web-animations/interfaces/Animation/commitStyles.html?label=master&label=stable&product=safari'>web platform tests</a>."
```

Comment:
I am removing this note in https://github.com/mdn/browser-compat-data/pull/7233. Review there much appreciated!

Reply:
I see this information came from https://github.com/mdn/browser-compat-data/pull/5963#discussion_r421349707